package.json

```
@@ -127,6 +127,7 @@
     "inquirer": "^0.12.0",
     "ioredis": "^4.3.0",
     "is-my-json-valid": "^2.19.0",
+    "jade": "~1.11.0",
```

Please use the `^` syntax 🙂

```suggestion
    "jade": "^1.11.0",
```